.rubocop.yml

```diff
@@ -13,6 +13,10 @@ Layout/BlockEndNewline:
   Exclude:
     - spec/features/search_catalog_spec.rb
 
+Performance/HashEachMethods:
+  Exclude:
+    - app/helpers/blacklight_helper.rb
```

Comment:
No need to disable this linter. It's telling you that since you don't use the variable `_field_name`, you should just use the `.each_value()` method instead of `.each`

see https://www.rubydoc.info/gems/rubocop/0.41.2/RuboCop/Cop/Performance/HashEachMethods